pkg/vault/contracts/VaultExtension.sol

```diff
@@ -36,6 +36,7 @@ import {
 } from "@balancer-labs/v3-solidity-utils/contracts/openzeppelin/ReentrancyGuardTransient.sol";
 import { FixedPoint } from "@balancer-labs/v3-solidity-utils/contracts/math/FixedPoint.sol";
 import { PackedTokenBalance } from "@balancer-labs/v3-solidity-utils/contracts/helpers/PackedTokenBalance.sol";
+import { BufferHelpers } from "@balancer-labs/v3-solidity-utils/contracts/helpers/BufferHelpers.sol";
```

Comment:
Can remove BufferHelpers (here and below) in VaultExtension; looks like they're used in the main Vault now. (Unless something that uses it gets moved back.)